dms/models/access_groups.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2017-2019 MuK IT GmbH
+# Copyright 2020 RGB Consulting
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class DmsAccessGroups(models.Model):
+    _name = "dms.access_groups"
```

Comment:
Avoid plurals in model names and optionally also `_`:
```suggestion
    _name = "dms.access.group"
```

Reply:
Yes, it implies some changes, but it has sense